microsoftgraph/microsoft-graph-explorer-v2
# Feature: preview opt out

## Overview
Adds a toggle to the sidebar to move from preview to classic
Also moves the authentication component to the sidebar, and fixes it for smaller screens

Fixes #165 
Fixes #213 

### Demo

Logged in desktop
![image](https://user-images.githubusercontent.com/53429535/66734206-de000c80-ee6a-11e9-871d-0f3c6808f04a.png)

Logged in mobile
![image](https://user-images.githubusercontent.com/53429535/66734317-4b13a200-ee6b-11e9-8e05-bfc62372039b.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Run GE Classic note via terminal
* After GE is running, run v2 via terminal
* When both are running go to v2 and switch the toggle. 
* Notice that it reloads the page with Classic instead of v2
